src/components/tooltip/CallNode.js

```diff
@@ -212,6 +215,45 @@ export class TooltipCallNode extends React.PureComponent<Props> {
       }
     }
 
+    // Finding current frame and parent frame URL(if there is).
+    let pageAndParentPageURL;
+    if (pages) {
+      const page = pages.find(p => p.innerWindowID === innerWindowID);
```

Comment:
Optional, or follow-up: I noticed in the codecov report that this area wasn't covered by a test. It'd be good to snapshot test, and maybe use `getByText` to find the URLs in the component.

Reply:
Adding tests for this, but probably it would be good to review those tests. Will open a follow-up PR for that.